src/DSP.jl

```diff
@@ -54,5 +54,12 @@ include("lpc.jl")
 using Reexport
 @reexport using .Util, .Windows, .Periodograms, .Filters, .LPC
 
+if isdefined(Base, :lastindex)
+    using Polynomials: Poly
+    if !applicable(Base.lastindex, Poly(0.0))
+        Base.lastindex(p::Poly) = endof(p)
```

Comment:
This is type piracy

Reply:
Sure. That's why the corresponding commit message says "to be deleted", which I'll now (well, on Monday, that is) do, as Polynomials has been fixed.

Reply:
Ah sorry, didn't read the commit messages. The Polynomials tag has been merged now, btw.